Production/govuk_ios/Views/Topics/TopicOnboardingCard.swift

```diff
@@ -0,0 +1,181 @@
+        imageView.tintColor = UIColor.govUK.fills.surfaceButtonPrimary
+        imageView.preferredSymbolConfiguration = config
+        imageView.setContentHuggingPriority(.defaultHigh, for: .vertical)
+        return imageView
+    }()
+
+    private lazy var cardStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.spacing = 8
+        stackView.alignment = .center
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        return stackView
+    }()
+
+    init(viewModel: TopicOnboardingCardModel) {
+        self.viewModel = viewModel
+        super.init(frame: .zero)
+        configureUI()
+        configureConstraints()
+        configureGestures()
+        configureAccessibility()
+    }
+
+    required init(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func configureUI() {
+        backgroundColor = UIColor.govUK.fills.surfaceCard
+        layer.borderWidth = 1
+        layer.cornerRadius = 10
+        layer.masksToBounds = true
+        layer.borderColor = UIColor.secondaryBorder.cgColor
+        addSubview(cardStackView)
+        selectedStackView.addArrangedSubview(selectedIcon)
+        selectedStackView.addArrangedSubview(selectedLabel)
+        cardStackView.addArrangedSubview(icon)
+        cardStackView.addArrangedSubview(titleLabel)
+        cardStackView.addArrangedSubview(descriptionLabel)
+        cardStackView.addArrangedSubview(selectedStackView)
+    }
+
+    private func configureConstraints() {
+        NSLayoutConstraint.activate([
+            cardStackView.topAnchor.constraint(
+                equalTo: safeAreaLayoutGuide.topAnchor,
+                constant: 16),
+            cardStackView.bottomAnchor.constraint(
+                equalTo: safeAreaLayoutGuide.bottomAnchor,
+                constant: -16),
+            cardStackView.leadingAnchor.constraint(
+                equalTo: layoutMarginsGuide.leadingAnchor,
+                constant: 8),
+            cardStackView.trailingAnchor.constraint(
+                equalTo: layoutMarginsGuide.trailingAnchor,
+                constant: -8)
+        ])
+    }
+
+    private func configureGestures() {
+        let tapGestureRecognizer = UITapGestureRecognizer(
+            target: self,
+            action: #selector(cardTapped)
+        )
+        self.addGestureRecognizer(tapGestureRecognizer)
+    }
+
+    private func configureAccessibility() {
+        self.isAccessibilityElement = true
+        self.accessibilityTraits = .button
+        self.accessibilityLabel = viewModel.title
+    }
+
+    @objc
+    private func cardTapped() {
+        viewModel.isSelected.toggle()
+        toggleTintColourOfCard()
+        toggleSelectedIconAndTextViews()
+        viewModel.tapAction(
+            viewModel.isSelected
+        )
+    }
+
+    private func toggleSelectedIconAndTextViews() {
+        let selectedColourTint = ColorResource(
```

Comment:
Can you rename instances of `colour` -> `color` please? Although not American, we are keeping consistency with the Apple APIs on this one to make searching easier, because of the number of word uses.